tests/read_frame.h

```diff
@@ -52,6 +52,15 @@ int read_frames(const std::string& filename) {
     return 1;
   }
 
+  // Reading specific entry
+  {
+    auto frame = podio::Frame(reader.readEntry("events", 4));
+    processEvent(frame, 4, reader.currentFileVersion());
+
+    auto otherFrame = podio::Frame(reader.readEntry("other_events", 4));
+    processEvent(otherFrame, 4 + 100, reader.currentFileVersion());
+  }
```

Comment:
Can you add two more tests here?
- Calling `readNextEntry` after a specific Frame has been read to "document" that it will continue from the previously read frame
- Calling `readEntry` again but with a smaller entry number to make sure that "jumping back" also works.

Reply:
Add :)